rasa/shared/utils/schemas/domain.yml

```diff
@@ -47,6 +47,8 @@ mapping:
       regex;([A-Za-z]+):
         type: "map"
         mapping:
+          regex;([A-Za-z]+):
```

Comment:
can't we just do `allowempty: True`? See docs here https://pykwalify.readthedocs.io/en/unstable/validation-rules.html#allowempty

Reply:
Indeed! I only discovered where these docs were after  the fact so I went off of the rest of the schema, but this makes more sense